### ISSUE TITLE
[Feature Request] Let lodestone docker create more dockers

### ISSUE THREAD

**Issue by Joly0** [Lodestone-Team/client#180]
Afaik as i see, lodestone does create game instances inside the same docker container that is running the "main" instance.

I think it would be great if lodestone could use the hosts docker engine to run the game instances as separate docker containers (just as an option). 
This might help with setting up new game instances, isolate different games further and help for later scalability

**Comment by CheatCod** [comment]
This can be done with atom https://github.com/Lodestone-Team/lodestone/wiki/Intro-to-Lodestone-Atom

**Comment by Joly0** [comment]
You are right, but this shouldnt need any additional configuration in atom or should atleast be a required setting for it, so any instance can be used with docker but it is not required to do so
